lang/pt.js

```diff
@@ -129,20 +129,20 @@ export default {
 	"components.table-controls.label": "Ações para a tabela",
 	"components.tabs.next": "Ir para frente",
 	"components.tabs.previous": "Ir para trás",
-	"components.tag-list.clear": "Clique em, pressione Backspace ou pressione a tecla Delete para remover o item {value}",
+	"components.tag-list.clear": "Clique, pressione a tecla Backspace ou a tecla Delete para remover o item {value}",
 	"components.tag-list.clear-all": "Limpar tudo",
-	"components.tag-list.cleared-all": "Todos os itens da lista de marcas foram removidos",
-	"components.tag-list.cleared-item": "Item {value} da lista de marcas removido",
+	"components.tag-list.cleared-all": "Todos os itens da lista de etiquetas foram removidos",
+	"components.tag-list.cleared-item": "Item {value} da lista de etiquetas removido",
 	"components.tag-list.interactive-label": "Lista de marcas, {count} itens",
 	"components.tag-list.num-hidden": "+ {count} mais",
 	"components.tag-list.role-description": "Lista de marcas",
 	"components.tag-list.show-less": "Mostrar menos",
-	"components.tag-list.show-more-description": "Selecione para mostrar itens ocultos da lista de marcas",
+	"components.tag-list.show-more-description": "Selecione para mostrar itens ocultos da lista de etiquetas",
 	"components.tag-list-item.role-description": "Marca",
 	"components.tag-list-item.tooltip-arrow-keys": "Teclas de seta",
-	"components.tag-list-item.tooltip-arrow-keys-desc": "Mover entre marcas",
-	"components.tag-list-item.tooltip-delete-key": "Retroceder/Excluir",
-	"components.tag-list-item.tooltip-delete-key-desc": "Excluir a marca de foco",
+	"components.tag-list-item.tooltip-arrow-keys-desc": "Mover entre etiquetas",
+	"components.tag-list-item.tooltip-delete-key": "Backspace/Delete",
```

Comment:
Looks like "Backspace" is used in other translations of keyboard keys in this file, so this change is consistent with that.